src/sass/_sections.scss

```diff
@@ -5,7 +5,10 @@ section,
   .inner {
     margin: 0 24px;
```

Comment:
We should replace this block of code with the section-inner-padding mixin in `src/sass/base/_mixins.scss` so we get standard padding for all `.inner` sections